gcsa/google_calendar.py

```diff
@@ -79,7 +92,7 @@ def _get_credentials(self):
 
         return credentials
 
-    def add_event(self, event):
+    def add_event(self, event, sendUpdates=SendUpdatesMode.NONE):
```

Comment:
Trying to pep8-ify the library, so I'd use snake case here (`send_updates`)

Reply:
Done!